test/recipe-test.js

```diff
@@ -507,4 +508,17 @@ describe('filterTagsByTagName', () => {
     
     expect(filteredTagData).to.deep.equal(expectedTagData);
   });
-});
\ No newline at end of file
+});
+
+describe('sortByHits', () => {
+  it('should sort a list of recipes by most hits', () => {
+    let recipes = [{hits: 2}, {hits: 1}, {hits: 3}]
+    let sortedRecipes = sortByHits(recipes)
+    assert.deepEqual(sortedRecipes, [{hits: 3}, {hits: 2}, {hits: 1}])
+  })
+  it('should not alter the original set of recipes', () => {
+    let recipes = [{hits: 2}, {hits: 1}, {hits: 3}]
+    sortByHits(recipes)
+    assert.deepEqual(recipes, [{hits: 2}, {hits: 1}, {hits: 3}])
+  })
```

Comment:
gahhh i prolly woulda forgotten sort is a mutator, bad ass

Reply:
but also, as a thought experiment, would we care if our fetched recipes got sorted? 🤔

Reply:
I don't think we would actually care if they DID get sorted, but I feel its better to prevent the side effect in case there are other consequences down the road that I am not thinking of